awx/api/templates/instance_install_bundle/install_receptor.yml

```diff
@@ -7,14 +7,11 @@
       user:
         name: "{{ receptor_user }}"
         shell: /bin/bash
-    - name: Enable Copr repo for Receptor
-      command: dnf copr enable ansible-awx/receptor -y
```

Comment:
Shouldn't this be removed from previous installed receptors?

Reply:
what do you mean exactly? The next receptor-collection version will install receptor via these releases by default https://github.com/ansible/receptor/releases

thus the Copr repo is no longer needed going forward

Reply:
I'm thinking of upgrades of previous installs.